scripts/package_building/build_artifacts.sh

```diff
@@ -474,6 +474,7 @@ function prepare_perf_rhel (){
     if [[ "$package_prefix" != "" ]];then
         sed -i -e "s/Name:.*/Name:  ${package_prefix}-perf/g" "SPECS/$spec"
     fi
+    check_spec_version "SPECS/$spec" "$kernel_version"
```

Comment:
check is a do nothing verb, and this method does also change the spec.

to keep consistency with the line below, you can use something like
change_perf_spec